Description: Correct bashisms in installed scripts
 Some of the installed scripts shbang /bin/sh, but have bashisms.
 Quick fix is to shbang /bin/bash
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: no
Last-Update: <2017-11-27>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/MpiApps/apps/run_allhfilatency
+++ b/MpiApps/apps/run_allhfilatency
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 #
 # Copyright (c) 2015-2020, Intel Corporation
--- a/MpiApps/apps/run_cabletest
+++ b/MpiApps/apps/run_cabletest
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # BEGIN_ICS_COPYRIGHT8 ****************************************
 #
 # Copyright (c) 2015-2020, Intel Corporation
